packages/version/src/conventional-commits/__fixtures__/fixed/scripts/erroring-preset.ts

```diff
@@ -4,5 +4,7 @@ const erroringPresets = {
   },
 };
 
+// not likely we support this
+// https://github.com/vitest-dev/vitest/issues/7920
 export default erroringPresets;
-module.exports = erroringPresets;
+// module.exports = erroringPresets;
```

Comment:
This combination breaks on vite-node (Vite doesn't even support `module.exports` at all). I don't think Vitest / vite-node should support this, so you probably need to fix this one on your side.

Reply:
yeah that should be fine, I'm building as ESM-Only for the past 2 major releases, so it should be fine to remove any old `module.exports` like this one, as long as the test still work though. It might need adjustments